### PR TITLE
Stricter Travis checks

### PR DESCRIPTION
**Pull request by seiflotfy**

### CODE REVIEW

File: src/skizze-cli/bridge/loop.go [hunk 1]
```diff
@@ -32,7 +32,8 @@ var typeMap = map[string]pb.SketchType{
 
 func setupClient() (pb.SkizzeClient, *grpc.ClientConn) {
 	// Connect to the server.
-	conn, err := grpc.Dial("127.0.0.1:3596", grpc.WithInsecure())
+	var err error
+	conn, err = grpc.Dial("127.0.0.1:3596", grpc.WithInsecure())
```
Comment (njpatel) [inline src/skizze-cli/bridge/loop.go]: Why this change? Was it further down?

Comment (seiflotfy) [inline src/skizze-cli/bridge/loop.go]: Above it conn was already defined as a global
